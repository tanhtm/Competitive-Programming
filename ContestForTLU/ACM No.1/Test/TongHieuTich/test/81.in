70
-241 541
341 668
838 476
987 391
-553 113
713 631
5 592
-967 631
171 526
545 174
1 111
501 -515
-980 -504
-134 215
806 -622
-252 -776
-653 -849
-374 -800
-911 662
-176 951
-454 -374
-588 211
833 -176
-303 -902
102 493
-645 788
801 -912
862 837
-210 32
595 683
-210 409
-311 -719
212 -380
-519 438
-701 -782
970 480
864 -186
-221 -868
-276 -97
-168 817
-548 -940
-377 -51
184 -382
-213 -275
461 11
-878 120
569 -108
-276 -496
435 403
-453 -625
6 -287
-561 825
835 -530
-2 -867
974 362
-209 474
-876 -313
289 -749
35 -160
-984 -860
742 373
-239 -152
-866 -24
497 -236
900 -874
974 69
-228 47
-907 -923
-647 316
-820 109
