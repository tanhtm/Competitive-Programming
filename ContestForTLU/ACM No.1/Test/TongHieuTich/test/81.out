-58650
-82485
118917
205322
73260
27552
-87609
134232
-61858
66687
-3080
-3556
176596
-7067
65688
-134668
-73598
-125031
97919
-218356
16560
75305
165728
-180448
-58161
-51229
-47535
10618
10769
-28116
-30795
-105060
-24864
19379
-30030
177625
177975
-176145
16691
-159816
-145824
34882
-28017
-7564
53100
189121
78024
-42460
6704
-46354
-20583
-91476
104081
-187921
204408
-45248
167351
-119370
-6093
57164
102858
8504
187345
47828
11531
235978
12443
-7320
79688
165129
